### PR TITLE
Fix/eslint

### PR DESCRIPTION
**Pull request by orrgottlieb**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+module.exports = {
+  globals: {
+    jest: "writeable",
+    describe: "readable",
+    it: "readable",
+    expect: "writeable",
+    afterEach: "writeable",
+    beforeEach: "writeable",
+    beforeAll: "readable",
+    afterAll: "readable"
```
Comment (sahariko) [inline .eslintrc.js:10]: These globals are relevant only for test files, and can be achieved more easily with:
```js
{
  overrides: [
    {
      files: [testFilePattern],
      environment: {
        jest: true
      }
    }
  ]
}
```

File: .eslintrc.js [hunk 2]
```diff
@@ -1,13 +1,32 @@
...
+  },
+  extends: ["plugin:react/recommended", "airbnb", "plugin:react-hooks/recommended"],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 13,
+    sourceType: "module"
+  },
+  plugins: ["import", "react", "jsx-a11y", "json", "markdown"],
```
Comment (sahariko) [inline .eslintrc.js:24]: I think we still want the `react-hooks` plugin.

Comment (orrgottlieb) [reply .eslintrc.js:24]: @sahariko 
image

Comment (sahariko) [reply .eslintrc.js]: Aaaah I see, since we're not extending anything, it's not needed.

**Review by sahariko — APPROVED** [review]

**Pull request merged**